src/backend/dx12/src/command.rs

```diff
@@ -1243,9 +1251,10 @@ impl com::RawCommandBuffer<Backend> for CommandBuffer {
         }
     }
 
-    fn bind_vertex_buffers(&mut self, vbs: pso::VertexBufferSet<Backend>) {
+    fn bind_vertex_buffers(&mut self, first_binding: u32, vbs: pso::VertexBufferSet<Backend>) {
         // Only cache the vertex buffer views as we don't know the stride (PSO).
-        for (&(buffer, offset), view) in vbs.0.iter().zip(self.vertex_buffer_views.iter_mut()) {
+        assert!(first_binding as usize <= MAX_VERTEX_BUFFERS);
```

Comment:
`<` should be enough

Reply:
Is it maybe enough to ignore range checks entirely and let the subslice indexing on the next line panic? I originally only added the range checking to keep the now-removed `vertex_buffers_first_slot` and `vertex_buffers_num_views` variables consistent with the `zip` behavior. If we're going to assert here, we should probably check `vbs.len()` as well which might be silently truncated (which was the behavior before this change).